### ISSUE TITLE
Getting a stat error for fontsettings.js

### ISSUE THREAD

**Issue by falc0nit3** [billryan/docker-gitbook#3]
When I run the serve command I get the following error:
# serve
docker run --rm -v "$PWD:/gitbook" -p 4000:4000 billryan/gitbook gitbook serve
Error: ENOENT: no such file or directory, stat '/gitbook/_book/gitbook/gitbook-plugin-fontsettings/fontsettings.js'

**Comment by billryan** [comment]
I will check GitBook changes this tonight or this weekend.

**Comment by billryan** [comment]
Did you try init first?

**Comment by falc0nit3** [comment]
Yeah I did. Tried in a clean new directory and get the same behavior. Thanks for looking into it though! Much appreciated.

**Comment by billryan** [comment]
Run install first, I forgot to update it in documents.
```
docker run --rm -v "$PWD:/gitbook" -p 4000:4000 billryan/gitbook gitbook install
```

Test successfully with my repo https://github.com/billryan/algorithm-exercise

**Comment by falc0nit3** [comment]
Umm getting this now:
Tried Install, init and then Serve and get this error then.
Error: ENOENT: no such file or directory, stat '/gitbook/_book/gitbook/gitbook-plugin-highlight/ebook.css'

I'll try again on a different machine and get back to you

**Comment by billryan** [comment]
You can paste your OS/docker environment and your gitbook repo address here and I will look into it.

**Comment by julio25frantz** [comment]
Did anyone get a solution?

**Comment by yeganehaym** [comment]
i have the same problem
